_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -56,65 +56,44 @@ This rule relies on browsers and assistive technologies to support leaving out [
 
 #### Passed Example 1
 
-Element has required states (no properties required for this role)
+This `checkbox` has the required property `aria-checked`.
 
 ```html
-<div role="option" aria-selected="false"></div>
+<div role="checkbox" aria-checked="false"><div></div></div>
```

Comment:
I'm not a huge fan of empty `div`s. Can it be removed without breaking the example? Or maybe replaced by something somewhat meaningful.
```suggestion
<div role="checkbox" aria-checked="false"></div>
```

Reply:
I don't actually know how that ended up like that. Thanks.